content/post/Start-up-dd.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Start-Up Deep Dive: MVP Building and User Generation "
+date: 2023-01-26T19:03:46+05:30
+authors: 
+    - nikita-ann-mathen
+Description: "A detailed summary of the fifth session of Startup Deep Dive on “MVP building” taken by our alumni."
+Tags: ["startup","startupdeepdive","MVP", "User Generation"]
+Categories: ["Startup"]
+DisableComments: false
+---
+
+A key ingredient to getting started on your entrepreneurial journey is finding the self-belief that it is possible for you to build an entire company from scratch. And what can be a more significant source of inspiration than hearing from alumni of your colleagues who have successfully achieved the same?
+
+
+Mr. Sandeep S and Ms. Ashwathy Venugopal are X-MECians and co-founders of Avasarshala, an ed-tech startup that curates talent-based opportunities for children. Ashwathy has even been recognized as one of the top 75 women entrepreneurs in India by the Central Government. Together, they joined us for the 5th session of IEDC MEC’s Entrepreneurial Training Series, Startup Deep Dive on  ‘MVP Building And User Generation’.
+
+
+So what is MVP? A Minimum Viable Product is a low-budget version of a product with just enough features to be used by early customers. The feedback obtained from the MVP can then be used to guide the development of the final product.
+
+
+_The MVP of a wedding cake would be a cupcake!_
+
+
+This was the first myth that was busted in the session. While an MVP is a small-scale version made with the least amount of money, it should also retain the functionalities of the final product you envision. For a wedding cake, the MVP might be a simple sponge cake without all the fancy decorations, or it could be a diagram showing the design and features that the final cake will have. However, it cannot be a cupcake which is an entirely different product.
+
+
+Avasarshala started as a WhatsApp group, which served as its MVP. Identifying the target audience is crucial at this stage. The next step is to charge a small fee for the product. By analyzing the percentage of clients who are willing to pay for it, we get a rough estimate of the demand for the final product. This is also essential in measuring the product's sustainability in the market.
```

Comment:
Should we make Avasarshala as a hyperlink to their website.?

Reply:
@adithyaanilkumar can you send the correct link heree.

Is it this one? 
https://www.avasarshala.com/